src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -54,4 +62,23 @@ public FindCommand parse(String args) throws ParseException {
 
         return new FindCommand(findpredicate);
     }
+
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
+        return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
+    }
+
+    private static boolean areThereBlanks(List<String> stringList) throws ParseException {
+        String NAME_FORMAT_REQUIREMENT = "There should not be any blanks in name.\n" + "If you are searching for "
+                + "'n/John Doe', split them into 'n/John' and 'n/Doe' instead.";
+        for (String s : stringList) {
```

Comment:
Also, might want to make this into a function that checks if there are blanks in the string, then return `true` or `false`. Error can be thrown in `execute` instead. Because now it seems a bit weird that the main purpose of this function is to throw an error lol. Also, in this case, your return value will never be `false`, and your return value is not used also, so returning boolean also seems a bit off. Just some thoughts ahahhahaah.

Reply:
o shit I forgot to change this. i changed the return type to void now. Btw wouldn't it be too late to throw it in the execute stage? I thought it would be better to throw it in the parsing stage